document_wopi/demo/ir_config_parameter.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="param_client" model="ir.config_parameter">
+            <field name="key">document_wopi.client</field>
+            <field name="value">https://docs.therp.nl</field>
```

Comment:
there's not so much to worry about, WOPI clients only allow predefined domains (one of the reasons we need the SSL dance), so the only place to actually use this is from OCA's runbot and Therp's installation.
And in case the tests take too much of our resources, I can revoke access.